<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup Scan Axes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="SingleMotorSetup" name="motor1"/>
     </item>
     <item>
      <widget class="SingleMotorSetup" name="motor2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="stop_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Force Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_point_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop After Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="damap_check">
     <property name="text">
      <string>Region contains DA map</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleMotorSetup</class>
   <extends>QGroupBox</extends>
   <header>scanwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
